src/scripts/syncBreaches.js

```diff
@@ -17,15 +17,16 @@ import { finished } from 'node:stream/promises';
 import { createWriteStream } from "node:fs";
 import { Readable } from 'node:stream';
 import os from 'node:os';
-import aws from 'aws-sdk';
+import { Upload } from "@aws-sdk/lib-storage";
+import { S3, DeleteObjectCommand } from "@aws-sdk/client-s3";
```

Comment:
`DeleteObjectCommand` can be deleted now

Reply:
Thanks for testing, deleted in e6c07ec67.